web/src/buddy_list.ts

```diff
@@ -62,6 +62,19 @@ function get_total_human_subscriber_count(
     return 0;
 }
 
+function get_non_participant_subscribers_count(
+    current_sub: StreamSubscription | undefined,
+    get_all_participant_ids: () => Set<number>,
+): number {
+    if (!current_sub) {
+        return 0;
+    }
+    const all_subscribers = new Set(peer_data.get_subscribers(current_sub.stream_id));
+    const bot_ids = people.get_bot_ids();
+    const bot_ids_set = new Set(bot_ids);
+    return all_subscribers.difference(get_all_participant_ids()).difference(bot_ids_set).size;
+}
```

Comment:
I think this function counts deactivated users who are subscribers; `peer_data.get_subscriber_count` does not. But I think that's the empty set, so maybe it's OK.

Reply:
@arthurzengg can you confirm this is the empty set?